Gemfile

```diff
@@ -81,4 +81,5 @@ end
 ENV['PUPPET_VERSION'].nil? ? puppetversion = '~> 6.0' : puppetversion = ENV['PUPPET_VERSION'].to_s
 gem 'puppet', puppetversion, :require => false, :groups => [:test]
 
+gem 'faraday', '>= 0.17.0',              :require => false
```

Comment:
I guess this didn't mean to slip in?

Reply:
Ah yes. I remove it. 
But only by adding this constraint i can produce `CHANGELOG.md`.